appverifier/src/main/java/com/android/mdl/appreader/fragment/DeviceEngagementFragment.kt

```diff
@@ -150,10 +150,12 @@ class DeviceEngagementFragment : Fragment() {
         } else {
             shouldRequestPermission()
         }
-        transferManager.setNdefDeviceEngagement(
-            NfcAdapter.getDefaultAdapter(requireContext()),
-            requireActivity()
-        )
+        val adapter = NfcAdapter.getDefaultAdapter(requireContext())
+        if (adapter != null)
+            transferManager.setNdefDeviceEngagement(
+                adapter,
+                requireActivity()
+            )
```

Comment:
Need braces otherwise LGTM

Reply:
hi David, thanks for your review. done. also, added a toast for giving a heads-up to the user like it was present before.